pkgs/stdenv/generic/setup.sh

```diff
@@ -422,50 +421,55 @@ fi
 
 
 substitute() {
-    local input="$1"
-    local output="$2"
+    local input=$1
+    local output=$2
```

Comment:
Agree with @dezgeg here. Shell quoting is such a pain and source of subtle bugs already; although it may be technically correct, why not be explicit when we can do so trivially? This particular gun might have a fancy high tech anti-foot-shot protection device in it, but I'd rather just point it away from my feet.